phpcount.php

```diff
@@ -135,8 +132,8 @@ public static function GetHits($pageID, $unique = false)
         }
         else
         {
-            die("Missing hit count from database!");
-            return false;
+            //die("Missing hit count from database!");
+            return 0;
```

Comment:
Hmm, wait, how does this distinguish "the row simply isn't present in the database yet" from "the connection to the database catastrophically failed, so we don't know what the hit count is"? In the former case returning 0 is correct, but in the latter case it should throw an exception.